kirby/api/context.py

```diff
@@ -0,0 +1,60 @@
+import os
+from smart_getenv import getenv
+import json
+from copy import deepcopy
+
+TYPES = {"str": str, "list": list}
+
+KIRBY_ENV_SIGNATURE = "KIRBY_ENV_SIGNATURE"
```

Comment:
just for the sake of it, maybe add a couple of underscore in front of the name, just to make clear this is not something to be messed with

Reply:
Added `__` in front of it